lib/keystore/keyring.go

```diff
@@ -83,6 +86,8 @@ func NewSr25519Keyring() (*Sr25519Keyring, error) {
 		}
 
 		who.Set(reflect.ValueOf(kp))
+
+		kr.Keys = append(kr.Keys, kp)
```

Comment:
We can preallocate the slice and then use the index instead of appending.
kr.Keys = make([]*sr25519.Keypair{}, v.NumField())
kr.Keys[i] = kp

Reply:
updated!